MIGRATION.md

```diff
@@ -86,6 +90,69 @@ yarn add @contentul/f36-button@beta
 
 ## Changes per component in v4
 
+### Asset
+
+The API of the Asset component has not changed.
```

Comment:
maybe mention that we changed some props to follow the code guidelines

Reply:
I compared v4 vs v3 storybooks, and I didn't find any difference 🤔

Reply:
Ohh my bad, is because the `How to migrate` section has some differences on the code, but the code you put there is actually from the `AssetCard`, we don't have a codemod for `Asset`, need to update that section them.

Reply:
oh, my bad, I'll fix it!